### ISSUE TITLE
paper: clarity of examples

### ISSUE THREAD

**Issue by corybrunson** [mingzehuang/latentcor#14]
In a similar spirit to #9, i'm more open to more than bare-bones content in the software paper, but i do think the paper should be structured differently than documentation—for readability rather than for reference. In particular, the two examples are explained not in exposition within the paper but in comments embedded with the code. This makes them hard to follow on their own and to compare (e.g. syntactically) to each other. The heat map of differences between the method is useful, for instance, but i think it would be easier to interpret with a short paragraph of text than with the code chunks used to generate it.

_Part of [this JOSS review](https://github.com/openjournals/joss-reviews/issues/3634)._

**Comment by irinagain** [comment]
We have rewritten the example to remove comments as much as possible and explain things in text.

**Comment by corybrunson** [comment]
Great, thanks! I will have a look and close the issue after the next PDF update at the review.

**Comment by corybrunson** [comment]
Hm, i see what you have done and i do think it improves the readability. Though it introduces a new problem: code chunks are included but it's not clear what their outputs look like. (I'm not sure, but i think the previous version included returned values from the function calls, as in a vignette.)

Based on other software papers, i can think of a couple of styles that might read better as an article:
1. For each of the two examples, include a single code chunk with all the essential steps (with or without outputs), and refer to the lines in that chunk as needed in the text.
2. Omit code chunks entirely and just mention inline what functions or options are being used, e..g "Applying the original estimation method, using the parameter setting `method = "original"`, gives an estimated latent correlation of ...".

Do those make sense? I do think either would work.

**Comment by irinagain** [comment]
We didn't have returned values in the paper before, only in vignette, because the paper has to be in .md (whereas the vignette is in .Rmd). We believe it's useful to have the code chunk for new users (even though they are still present in vignette). We followed the option 1 above where we consolidated code chunks back into a single one for each example, and have an extended paragraph beforehand commenting on the chunk. Let us know if this works. Thanks!
